tracing-subscriber/src/filter/targets.rs

```diff
@@ -277,6 +277,31 @@ impl Targets {
         self
     }
 
+    /// Returns the explicitly set default level for this filter, if any.
+    ///
+    /// If a default level was set explicitly using [`with_default`](Self::with_default), that level
+    /// will be returned. Otherwise `None` will be returned.
+    ///
+    /// A return value of `None` is behaviourly equivalent to [`LevelFilter::OFF`], but
+    /// distinguishes between an explicitly set default level or the default one. If you only care
+    /// about the behaviour you can use `unwrap_or`:
+    ///
+    /// ```
+    /// use tracing_subscriber::filter::{LevelFilter, Targets};
+    ///
+    /// let filter = Targets::new();
+    /// let default_level = filter.default_level().unwrap_or(LevelFilter::OFF);
+    ///
+    /// assert_eq!(default_level, LevelFilter::OFF);
+    /// ```
```

Comment:
it might be nice to have an example that explicitly demonstrates when `None` is returned, and when `Some(LevelFilter::OFF)` is returned?

Reply:
I've bulked out the examples, perhaps excessively, to cover `with_default`/`parse` for all of set/unset/set to `OFF`.